pkg/cmd/runsGet.go

```diff
@@ -99,6 +99,12 @@ func (cmd *RunsGetCommand) createCobraCommand(
 	runsGetCobraCmd.PersistentFlags().StringVar(&cmd.values.result, "result", "", "A filter on the test runs we want information about. Optional. Default is to display test runs with any result. Case insensitive. Value can be a single value or a comma-separated list. For example \"--result Failed,Ignored,EnvFail\"")
 	runsGetCobraCmd.PersistentFlags().BoolVar(&cmd.values.isActiveRuns, "active", false, "parameter to retrieve runs that have not finished yet.")
 
+	runsGetCobraCmd.MarkFlagsMutuallyExclusive("name", "age")
```

Comment:
--name and --age may not be mux. These names aren't always unique. There may be >1 instance of a run with the same name. Being able to narrow it by age might be useful.

Reply:
huh, ok :)

Reply:
would it not be best to have the others not be mutually exclusive as well then?

Reply:
I think adding these constraints is fine/helpful.
If a test is re-queued (by the Galasa system), then you can end up with >1 runs.
The reqeustor will be the same in that case, so having that work with the name isn't helpful. The active flag is useless if you have the name.

I think that if we add constraints, we should beef-up the help to say `this flag can not be used with the xxx flag` ...etc.

Reply:
I agree with that, and for other commands too